wren-ai-service/eval/README.md

```diff
@@ -68,6 +68,58 @@ The evaluation results will be presented on Langfuse as follows:
 
 ![shallow_trace_example](../docs/imgs/shallow_trace_example.png)
 
+
+## How to use Dspy in Wren AI
+### Step 1: Generate evaluation dataset
+
+Please use eval.py and the spider2 v1 dataset to train an optimized dspy module (https://github.com/taoyds/spider/tree/master/evaluation_examples/examples)
```

Comment:
- what is `eval.py`?
- should be Spider v1 dataset
- is there relevant command here?